python/ola/Makefile.mk

```diff
@@ -104,4 +104,5 @@ CLEANFILES += \
     python/ola/ClientWrapperTest.sh \
     python/ola/OlaClientTest.sh \
     python/ola/PidStoreTest.sh \
-    python/ola/RDMTest.sh
+    python/ola/RDMTest.sh \
+    python/ola/__pycache__/*
```

Comment:
.gitignore too?

Does our other python code in tools/rdm need the same addition too?

Reply:
They are just pyc files in there so gitignore already covers them.  

tools/rdm will need a similar change, but it doesn't vaguely run under py3, so not an issue at present, and I think should be left for that change.  I had probably most of those changes done with the earlier pass, but no way to test them.

Reply:
I didn't know if we needed to ignore the folder too, or will git ignore it if it's ignoring all the files so it's empty (hence .gitkeep elsewhere)?

Ah I didn't realise tool/rdm was really broken too. We should be able to test against the dummy responders mostly (to prove it runs and hopefully gives correct results). I suspect it can wait until 0.11.0, TBH I generally suggest people run the tests against master for all the latest and greatest anyway.

Reply:
git (and distcheck) seem OK with empty directories